riscv/tests/common/mod.rs

```diff
@@ -1,45 +1,52 @@
 use mktemp::Temp;
-use powdr_number::GoldilocksField;
+use powdr_number::{BabyBearField, FieldElement, GoldilocksField, KnownField};
 use powdr_pipeline::{
-    test_util::{run_pilcom_with_backend_variant, BackendVariant},
+    test_util::{run_pilcom_with_backend_variant, test_plonky3_pipeline, BackendVariant},
     Pipeline,
 };
-use powdr_riscv::Runtime;
+use powdr_riscv::CompilerOptions;
 use std::{
     path::{Path, PathBuf},
     process::Command,
 };
 
 /// Like compiler::test_util::run_pilcom_asm_string, but also runs RISCV executor.
-pub fn verify_riscv_asm_string<S: serde::Serialize + Send + Sync + 'static>(
+pub fn verify_riscv_asm_string<T: FieldElement, S: serde::Serialize + Send + Sync + 'static>(
```

Comment:
Unfortunately we do need T around here because of Pipeline and inputs.

Reply:
either that or passing a `KnownField` argument?